### PR TITLE
Grammar changes

### PR DESCRIPTION
**Pull request by fredster33**

### CODE REVIEW

**Review by slawekjaranowski — CHANGES_REQUESTED** [review]
please also look at `src/site/markdown/index.md.vm`

File: README.md [hunk 1]
```diff
@@ -4,17 +4,17 @@
 [![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=org.simplify4u%3Aslf4j-mock&metric=alert_status)](https://sonarcloud.io/dashboard?id=org.simplify4u%3Aslf4j-mock)
 [![Lines of Code](https://sonarcloud.io/api/project_badges/measure?project=org.simplify4u%3Aslf4j-mock&metric=ncloc)](https://sonarcloud.io/dashboard?id=org.simplify4u%3Aslf4j-mock)
 
-Yet another way to testing logging in application which use SLF4J.
+Yet another way to test logging in to applications which use SLF4J.
```
[the remark this answers is not in the document]
Comment (fredster33) [reply README.md:7]: I believe that `to` is required, because `to` addresses the verb `test`.

[the remark this answers is not in the document]
Comment (slawekjaranowski) [reply README.md:7]: I think about: `logging in -> to <- applications` - but I'm not native speaker so I can wrong

[the remark this answers is not in the document]
Comment (fredster33) [reply README.md:7]: Could you clarify please?

[the remark this answers is not in the document]
Comment (slawekjaranowski) [reply README.md:7]: I see two options:
1. Yet another way to test logging in to applications which use SLF4J.
2. Yet another way to test logging in applications which use SLF4J.

why 1 is better than 2?

[the remark this answers is not in the document]
Comment (fredster33) [reply README.md:7]: "Logging in" requires a subject to apply to, and "to" I think is the transition to the subject.

[the remark this answers is not in the document]
Comment (slawekjaranowski) [reply README.md:7]: Here `logging` is used as process which produce trace/debug message, some system log and this is happened `in` application. 

We don't thing about `loging in` as changing permission to have access to application.

[the remark this answers is not in the document]
Comment (fredster33) [reply README.md]: All right, undid my edit for that section.

**Comment by slawekjaranowski** [comment]
please also look at `src/site/markdown/index.md.vm`

**Pull request closed without merging**